scribble-doc/scribblings/scribble/acmart.scrbl

```diff
@@ -0,0 +1,157 @@
+#lang scribble/manual
+@(require "utils.rkt" (for-label (except-in scribble/acmart title author)))
+
+
+@title{ACM Paper Format}
+
+@defmodulelang[scribble/acmart]{The @racketmodname[scribble/acmart]
+language is like @racketmodname[scribble/base], but configured with
+Latex style defaults to use the @filepath{acmart.cls} class
+file that is included with Scribble.}
+
+@(define-syntax-rule (defacmartid name)
+   @defidform[name]{
+Enables the @tt[(symbol->string 'name)] option. Use @racket[name] only on the
+same line as @hash-lang[], with only whitespace (or other options) between
+@racketmodname[scribble/acmart] and @racket[name]:
+@verbatim[#:indent 2]{
+  #lang scribble/acmart @"@"@(symbol->string 'name)
+}})
+
+@defacmartid[manuscript]
+@defacmartid[acmsmall]
+@defacmartid[acmlarge]
+@defacmartid[acmtog]
+@defacmartid[sigconf]
+@defacmartid[siggraph]
+@defacmartid[sigplan]
+@defacmartid[sigchi]
+@defacmartid[sigchi-a]
+@defacmartid[review]
+@defacmartid[screen]
+@defacmartid[natbib]
+@defacmartid[anonymous]
+@defacmartid[authorversion]
+
+The @racket[manuscript], @racket[acmsmall], @racket[acmlarge], 
+@racket[acmtog], @racket[sigconf], @racket[siggraph], @racket[sigplan],
+@racket[sigchi], and @racket[sigchi-a] options are all mutually exclusive,
+but may each be used in combination of @racket[review], @racket[screen], @racket[natbib],
+@racket[anonymous], and @racket[authorversion].
+
+
+@defproc[(abstract [pre-content pre-content?] ...) block?]{
+
+Generates a @tech{nested flow} for a paper abstract.}
+
+@defform[(include-abstract module-path)]{
+
+Similar to @racket[include-section], but incorporates the document in the
+specified module as an abstract. The document must have no title or
+sub-parts.}
+
+@defproc[(subtitle [pre-content pre-content?] ...) content?]{
+
+Use as the last argument to @racket[title] to specify a subtitle.}
+
+@defproc[(maketitle) block?]{
+
+Issues the @tt{\maketitle} command.  This must appear after the abstract
+any several other top-matter commands.  (See the acmart latex documentation.)}
+
+
+@deftogether[(
+@defproc[(acmJournal [journal pre-content?] ...) content?]
+@defproc[(acmConference [name pre-content?] [date pre-content?] [venue pre-content?]) content?]
+@defproc[(acmVolume [content pre-content?] ...) content?]
+@defproc[(acmNumber [content pre-content?] ...) content?]
+@defproc[(acmArticle [content pre-content?] ...) content?]
+@defproc[(acmYear [content pre-content?] ...) content?]
+@defproc[(acmMonth [content pre-content?] ...) content?]
+@defproc[(acmArticleSeq [content pre-content?] ...) content?]
```

Comment:
It's easy to write strings in scribble and doesn't require remembering the right type to use.  This interface is more consistent with the latex interface _and_ the existing `sigplan` library.